layouts/header.html

```diff
@@ -7,7 +7,7 @@
        </div>
      </div>
    <a href="https://www.bandwidth.com" class="aimg"><div class="headerLogo"></div></a>
-   <h2><a href="http://bandwidth-docs.s3-website-us-east-1.amazonaws.com/">Docs for Developers</a> &nbsp;&#8250;&nbsp; Bandwidth APIs</h2>
+   <h2><a href="https://new.dev.bandwidth.com">Docs for Developers</a> &nbsp;&#8250;&nbsp; Bandwidth APIs</h2>
```

Comment:
Does this cover https://bandwidth.codefactori.com/browse/DX-555? Just want to track what else is completed by this change

Reply:
yes

Reply:
Cool. Let's make sure to close it out when this PR is merged